packages/cryptoassets/data/trc20.js

```diff
@@ -4,5 +4,6 @@ module.exports = [
   ["TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9","JST","JUST GOV","TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9",18],
   ["TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT","USDJ","JUST Stablecoin","TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT",18],
   ["TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS","KLV","Klever","TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS",6],
-  ["TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh","SEED","SEED","TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh",6]
+  ["TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh","SEED","SEED","TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh",6],
+  ["TZJmk51TP3YEETw7phRP581A7H3VzLDFsa","TEUR”,”EUR AlteWay Token","TZJmk51TP3YEETw7phRP581A7H3VzLDFsa",4]
```

Comment:
these is a problem with this code, please use a code editor. there are `”` characters used instead of `"`